--- conf/nginx.conf.orig	2010-03-26 12:08:41.000000000 +0300
+++ conf/nginx.conf	2010-03-26 12:09:16.000000000 +0300
@@ -18,6 +18,10 @@
     include       mime.types;
     default_type  application/octet-stream;
 
+    # Don't forget set correct path to passenger dir!
+    # passenger_root = /opt/ree/lib/ruby/gems/1.8/gems/passenger-2.2.7;
+    # passenger_ruby = /opt/ree/bin/ruby;
+
     #log_format  main  '$remote_addr - $remote_user [$time_local] $request '
     #                  '"$status" $body_bytes_sent "$http_referer" '
     #                  '"$http_user_agent" "$http_x_forwarded_for"';
